app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -72,7 +72,7 @@
     public static final String ERROR_CODE = "Error Code";
     private static final String NAV_ITEM = "navItem";
     private static final String BOOKMARK = "bookmarks";
-
+	public static final String APP_LINK = "https://app_link_goes_here.com";
 
```

Comment:
Please move this [here](https://github.com/fossasia/open-event-android/blob/master/app/src/main/java/org/fossasia/openevent/api/Urls.java). We don't want organisers to open 10 files and edit the url's while getting the app ready for their event.